packages/react-reconciler/src/ReactFiberThenable.js

```diff
@@ -194,8 +194,9 @@ export function trackUsedThenable<T>(
           // this case include forcing a concurrent render, or putting the whole
           // root into offscreen mode.
           throw new Error(
-            'async/await is not yet supported in Client Components, only ' +
-              'Server Components. This error is often caused by accidentally ' +
+            'An unknown Component is an async Client Component. ' +
```

Comment:
Alternative would be "One of your Client Components is async". My concern for that wording is that it sounds like we do know which one but won't tell. "Unknown" makes it clearer that we just don't know. No strong opinion either way.